sympy/physics/units/definitions/unit_definitions.py

```diff
@@ -114,6 +108,21 @@
 becquerel = Bq = Quantity("becquerel", abbrev="Bq")
 
 
+# Common mass units
+
+mg = milligram = milligrams = Quantity("milligram", abbrev="mg")
+mg.set_global_relative_scale_factor(milli, gram)
+
+ug = microgram = micrograms = Quantity("microgram", abbrev="ug", latex_repr=r"\mu\text{g}")
+ug.set_global_relative_scale_factor(micro, gram)
+
+Da = dalton = Quantity("dalton", abbrev="Da")
+Da.set_global_relative_scale_factor(Rational(1.66, 10**(27)), gram)
```

Comment:
I think it should be 10**-27 (and many more decimal digits for 1.66). https://en.wikipedia.org/wiki/Dalton_(unit) (and note that 10**-27 is in kg.

Reply:
Compare to https://github.com/sympy/sympy/blob/0e4bab73562d6f90fcdf5fa079731f26a455347c/sympy/physics/units/systems/si.py#L165-L166 which is 1 Da.

Reply:
Thanks .Will make these changes quick !

Reply:
Is there a way to refer the Da to the amu so we only have the constant in one place?